scripts/peek_dma.py

```diff
@@ -0,0 +1,50 @@
+def main():
+    DMA1_BASE = 0x40020000
+    DMA2_BASE = 0x40020400
+    base = DMA1_BASE
+
+    cl = cmdline.split()
+    if len(cl) > 1:
+        dma = int(cl[1])
+        if dma == 1:
+            base = DMA1_BASE
+        elif dma == 2:
+            base = DMA2_BASE
+        else:
+            print("Bad DMA number, should be 1 or 2")
+            return
+
+    reg = Peek32(base, 43)
```

Comment:
Ah, this is expected to be run by the debug script itself, that's where e.g. Peek32 is coming from?

Right now the context that this is running in is tightly coupled to another script in another file.  This makes it impossible to use tools like `pytype` to check the code.  It also makes the code extremely difficult to test.

I would prefer if we accomplished this some other way.  Can this just be part of the main debug script?

Reply:
I hate to get this PR bogged down with this little debug script that no one else will likely ever use.  This really has nothing to do with the PR.  I only checked it in because Eugene had asked for an example of the scripts that can be run from the debug console back when he reviewed the debug stuff.
The debug scripts run from within the context of the debug console and are handy add-ons that can be used for all sorts of things.  When I'm debugging hardware I often make a script to allow me to quickly look at the registers to see if there's some issue.  I wrote this when I was trying to get the DMA working for the stepper driver and checked it in because I figured it might be handy for someone in the future.
I don't think this should be added to the debug console itself.  It's too obscure and adding things like this would just cause bloat in the console.  Maybe they don't belong checked in to git at all.  I could just keep this as part of my local toolbox since few other people will want to peek at DMA registers.